polkadot/runtime/rococo/Cargo.toml

```diff
@@ -67,7 +67,7 @@ pallet-scheduler = { path = "../../../substrate/frame/scheduler", default-featur
 pallet-session = { path = "../../../substrate/frame/session", default-features = false }
 pallet-society = { path = "../../../substrate/frame/society", default-features = false, features = ["experimental"] }
 pallet-sudo = { path = "../../../substrate/frame/sudo", default-features = false }
-frame-support = { path = "../../../substrate/frame/support", default-features = false }
+frame-support = { path = "../../../substrate/frame/support", default-features = false, features = ["tuples-96"] }
```

Comment:
I had to add this feature because the runtime pallets in `Rococo` exceeded 64. After adding this line, the tests started to pass.